src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -320,6 +320,38 @@ public static boolean isOlderThanAYear(Date compareDate) {
         return (differenceInDays > 365);
     }
     
+    /**
+     * Checks if the time falls between the time range specified.
+     * Start time is inclusive (i.e. {@code startTime <= time < endTime})
+     * @param startTime the start time of the time range
+     * @param endTime the end time of the time range
+     * @param time the time to be checked
+     * @return true if the time falls between the start and end time
+     */
+    public static boolean isStartTimeWithinRange(Date startTime, Date endTime, Date time) {
+        if (startTime == null || endTime == null || time == null) {
+            return false;
+        }
+        
+        return startTime.getTime() <= time.getTime() && time.getTime() < endTime.getTime();
+    }
+    
+    /**
+     * Checks if the time falls between the time range specified.
+     * End time is inclusive (i.e. {@code startTime < time <= endTime})
+     * @param startTime the start time of the time range
+     * @param endTime the end time of the time range
+     * @param time the time to be checked
+     * @return true if the time falls between the start and end time
+     */
+    public static boolean isEndTimeWithinRange(Date startTime, Date endTime, Date time) {
+        if (startTime == null || endTime == null || time == null) {
+            return false;
+        }
+        
+        return startTime.getTime() < time.getTime() && time.getTime() <= endTime.getTime();
+    }
```

Comment:
This method is to retrieve all open session within the certain time period. For example, if I want to find open feedback sessions between 1st April to 5th April, any feedback session that starts on 1st April to before 5th April or any feedback session that ends after 1st April to 5th April should qualify.

i.e.
start time >= 1st April and < 5th April (1st April <= start time < 5th April) or 
end time > 1st April and <= 5th April (1st April < end time <= 5th April)
should qualify. Hence the different in the return statement. @damithc